controllers/C_Prescription.class.php

```diff
@@ -60,17 +60,20 @@ function __construct($template_mod = "general")
             // $res = sqlStatement("SELECT * FROM drugs ORDER BY selector");
 
             $res = sqlStatement("SELECT d.name, d.ndc_number, d.form, d.size, " .
-                "d.unit, d.route, d.substitute, t.drug_id, t.selector, t.dosage, " .
-                "t.period, t.quantity, t.refills, d.drug_code " .
-                "FROM drug_templates AS t, drugs AS d WHERE " .
-                "d.drug_id = t.drug_id ORDER BY t.selector");
+            "d.unit, d.route, d.substitute, t.drug_id, t.selector, t.dosage, " .
+            "t.period, t.quantity, t.refills, d.drug_code, i.on_hand,i.expiration " .
+            "FROM drug_templates AS t " .
+            "JOIN drugs AS d ON d.drug_id = t.drug_id " .
+            "JOIN drug_inventory AS i ON d.drug_id = i.drug_id " .
```

Comment:
With inhouse_pharmacy will every drug record have a drug_inventory record?  I haven't used the inhouse_pharmacy module much so you'll need to confirm if this is true or not.  If drug records can exist without a drug_inventory record should make the drug_inventory a LEFT JOIN as you will break existing data records for other users.

Reply:
The medications whose template has been created only those show up in drop down menu. If we have added the drug details but not created any template for it then those drugs dont show up in the drop down by default.
( i feel this is a small bug because the medicine should anyway appear in the drop down , if it has been added to the inventory, irrespective of a template/s having been created or not for it.

This is applicable to only the drugs in the pharmacy drug_record. So i do not think there will be any issue.
Only the drugs listed in drug inventory show up in the drop down.